packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -22,6 +23,11 @@ type ChatMessageComponentProps = {
    * Setting to false will mean they are on a new line inside the editable chat message.
    */
   inlineAcceptRejectEditButtons: boolean;
+  /**
+   * Optional callback to render uploaded files in the message component.
+   * @beta
```

Comment:
no need to add a beta tag here since `ChatMessageComponentProps` are internally used.